mozjs/python/mozbuild/mozbuild/test/backend/data/host-rust-library-features/Cargo.toml

```diff
@@ -1,15 +0,0 @@
-[package]
```

Comment:
Same question - do we need to automate the removal of these each time we update SpiderMonkey?